### DISCUSSION TITLE
How can I change document title on client?

### DISCUSSION THREAD

**Discussion by arvinxx** [vercel/next.js#56280] [App Router]
AppRouter metadata is only work on rsc. and next/head is not working with app Router.

How can I change document title on client?

**Comment by icyJoseph** [comment]
> How can I change document title on client?

Assuming you just want to change it once in the client, you can just do a `document.title = "something something'`, preferably within a use effect or in response to some user interaction.

Assuming you want to have a `use client` page, you could use layout.tsx as sibling and use the metadata there (preferred), OR refactor so that the page.tsx is a server component, that does the metadata, and renders as its only child, a `use client` component (which uses to be the page).
